src/main/java/task/Event.java

```diff
@@ -0,0 +1,39 @@
+package task;
+
+import java.time.LocalDateTime;
+
+public class Event extends Task {
+    LocalDateTime startDate;
+    LocalDateTime endDate;
+
+    public Event(String taskName, LocalDateTime start, LocalDateTime end) {
+        super(taskName);
+        this.startDate = start;
+        this.endDate = end;
+    }
+
+    /**
+     * Converts the LocalDateTime object to a cleaner looking string.
+     * Similar to Deadline::dateTimeToCleanString
+     * 
+     * @param dt the LocalDateTime object to be converted
+     * @return the date and time formatted like "10:30pm on AUGUST 29, 2024"
```

Comment:
Should there be periods at the end of these lines?

Reply:
Yes, thanks!